### DISCUSSION TITLE
v1.6.0-rc.5 - arch

### DISCUSSION THREAD

**Discussion by paesiv** [evilsocket/opensnitch#876] [Releases]
hi, is it possible to get latest [v1.6.0-rc.5](https://github.com/evilsocket/opensnitch/releases/tag/v1.6.0-rc.5) build on yay or instructions how to build it for arch?

thanks

**Comment by gustavo-iniguez-goya** [comment]
hi @paesiv ,

Yes, you need to clone the AUR repo:
```bash
$ git clone https://au.archlinux.org/opensnitch-git.git
$ cd opensnitch-git
```

then edit the line 39 of PKGBUILD: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=opensnitch-git#n39
and change `utils.py` by `utils/__init__.py`

then execute makepkg -si: ` ~/opensnitch-git $ makepkg -si`

**Reply by paesiv** [reply]
thank you

after install it says 1.6.0rc2, how to install latest?

**Reply by gustavo-iniguez-goya** [reply]
Execute `opensnitchd -version` and `opensnitch-ui --help` to see the versions you have currently installed.

While compiling latest sources it should have printed out: "Making package: opensnitch-git 1.6.0rc5" 

If it still says 1.6.0rc2  after `makepkg -si`, execute sudo `makepkg -i` (I'm not an arch user btw O:) so maybe you should better ask for help on the AUR forums)
